source/extensions/filters/http/jwt_authn/authenticator.h

```diff
@@ -9,6 +9,8 @@
 #include "jwt_verify_lib/check_audience.h"
 #include "jwt_verify_lib/status.h"
 
+#include "absl/time/clock.h"
```

Comment:
Do we have any regression or unit/integration tests for this?

Reply:
Tests? Jeeze. What roughnecked thinking. No, seriously. This is my first contribution here and I wasn't actually planning on going through by myself. Anyways, I don't mind at all doing it but need to get set up with a proper build environment to do real work. The coming weekend seems a good time to do so.

Reply:
Ah heck, here I am working already.

Reply:
This header is not needed any more. please remove it